test/EFCore.PG.FunctionalTests/NpgsqlMigrationSqlGeneratorTest.cs

```diff
@@ -726,20 +726,6 @@ public void CreatePostgresEnum()
             Assert.Equal(@"CREATE TYPE public.my_enum AS ENUM ('value1', 'value2');" + EOL, Sql);
         }
 
-        [Fact]
-        public void CreatePostgresEnumWithSchema()
-        {
-            var op = new AlterDatabaseOperation();
-            PostgresEnum.GetOrAddPostgresEnum(op, "some_schema", "my_enum", new[] { "value1", "value2" });
-            Generate(op);
-
-            Assert.Equal(
-                @"CREATE SCHEMA IF NOT EXISTS some_schema;" + EOL +
-                @"GO" + EOL + EOL +
-                @"CREATE TYPE some_schema.my_enum AS ENUM ('value1', 'value2');" + EOL,
-                Sql);
-        }
-
```

Comment:
There's a test for `CreatePostgresEnum()` above, this one was only needed to verify the `EnsureSchemaOperation` and can now be safely removed.